src/libs/Navigation/CustomActions.js

```diff
@@ -24,7 +24,19 @@ function pushDrawerRoute(screenName, params, navigationRef) {
 
         if (activeReportID === params.reportID) {
             if (state.type !== 'drawer') {
-                navigationRef.current.dispatch(StackActions.pop());
+                navigationRef.current.dispatch(() => {
+                    // If there are multiple routes then we can pop back to the first route
```

Comment:
This code looks familiar... 😄 

Are you OK with just copying and pasting things from other places? 
Or should we think of a better way to handle this duplication?

Reply:
Sorry about that. I just found this to be straight to start with this. I can definitely refactor it into a function. :smile:

Reply:
I am not sure the best way to dry this up. I can basically call the `DismissModal` here instead. It does basically the same thing. What do you suggest?

Reply:
Please take some time to look into it and propose some options?

Reply:
Sure. Thanks.